README.md

```diff
@@ -65,6 +65,8 @@ Rematch is Redux best practices without the boilerplate. No more action types, a
   * [Immer](./plugins/immer/README.md)
 * [Inspiration](./docs/inspiration.md)
 
+#### [中文手册](https://rematch.gitbook.io/handbook)
```

Comment:
Could we change this as a translation section, in order to perhaps support other languages?

```
##### Translations
* [中文手册](https://rematch.gitbook.io/handbook)
```

Reply:
nice